scripts/dependencies/from_package.py

```diff
@@ -18,19 +18,24 @@
 import re
 import sys
 
-pattern = re.compile(u'^{}'.format(sys.argv[1]))
+package_path = sys.argv[1]
+if package_path.endswith('/'):
+    package_path = package_path[:-1]
```

Comment:
I'm not sure we actually want to strip the trailing slash; that makes it hard, for example, to find dependencies from "openedx/core/djangoapps/content" but not "openedx/core/djangoapps/contentserver"

Reply:
None of the output roots or names contain a trailing slash so it was preventing things from getting matched.

Reply:
Hmm, `scripts/dependencies/from_package.py openedx/core/djangoapps/content/` produces the output I would expect without this change.